src/i18n/en-US/governanceOverview.ts

```diff
@@ -61,7 +61,7 @@ const governanceOverview = {
     {
       heading: 'Decision and next steps',
       content:
-        'If your Business Case is approved, you will receive a unique Life Cycle ID. If it is not approved, you will recieve documented next steps or concerns to address in order to proceed.'
+        'If your Business Case is approved, you will receive a unique Life Cycle ID. If it is not approved, you will receive documented next steps or concerns to address in order to proceed.'
```

Comment:
oof, was this me?  :panik:

Reply:
Ugh, it was me. I really hope I copied this text from Figma so I can blame someone else 🫠

Reply:
haha - doesn't matter who. i'm sure no one ever noticed it